src/Components/Server/src/Circuits/RenderBatchWriter.cs

```diff
@@ -5,9 +5,15 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text;
+#if !IGNITOR
```

Comment:
This is kinda gross. This namespace doesn't exist in Ignitor, but I could make an empty one if that's cleaner?

Reply:
meh. I think it's ok.